src/schema.rs

```diff
@@ -0,0 +1,7 @@
+table! {
+    users (id) {
+        id -> Int4,
+        name -> Text,
+        age -> Int4,
+    }
+}
```

Comment:
Do we need this for the tests? 🤔

Reply:
Nope! Removed.

Guess that was one of the potentially unwanted things diesel setup does...